src/Psalm/Internal/Provider/FunctionReturnTypeProvider.php

```diff
@@ -79,7 +78,6 @@ public function __construct()
         $this->registerClass(ArraySpliceReturnTypeProvider::class);
         $this->registerClass(ArrayReverseReturnTypeProvider::class);
         $this->registerClass(ArrayUniqueReturnTypeProvider::class);
-        $this->registerClass(ArrayValuesReturnTypeProvider::class);
```

Comment:
It's redundant now, right?

Reply:
Right, I removed the return type provider and added it as a stub. 

https://github.com/vimeo/psalm/pull/7396/files#diff-483b988f07809c751ac53f905f9b4dc8b0093e9af15578ee3ac009b9dfe394b5R170-R181